hypothesis-python/tests/cover/test_lookup.py

```diff
@@ -102,6 +102,7 @@ def test_typing_Type_Union(ex):
         (typing.List[int], list, int),
         (typing.Tuple[int], tuple, int),
         (typing.Tuple[int, ...], tuple, int),
+        (typing.Iterable[int], typing.Iterable, int),
```

Comment:
Isn't it the same as the one 3 lines below? What's the difference?

Reply:
It is... and there isn't any.

Turns out that this is more subtle than I thought, and the existing tests only pass because they happen to use `int` :sweat_smile:

Reply:
That's why you can't review things based on a small diff 😅

Reply:
Haha, and I was so confident too... it all worked on my machine!  (for the wrong reasons, of course :facepalm:)